packages/fxa-shared/l10n/branding.ftl

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### Localization for fxa
+
+## Brand and product terms to be used globally
+
+# Note: Terms should not be transliterated or translated
+# Declension should be avoided
+
+# Firefox and Mozilla Brand
+-brand-name-mozilla = Mozilla
+-brand-name-firefox = Firefox
+
+# Mozilla and Firefox products that should be treated as brands (not translated)
+-product-name-mozilla-vpn = Mozilla VPN
+-product-name-firefox-cloud = Firefox Cloud
+-product-name-firefox-monitor = Firefox Monitor
+-product-name-firefox-relay = Firefox Relay
+
+# Other brands
+-brand-name-apple = Apple
+-brand-name-google = Google
+-brand-name-paypal = PayPal
+-brand-name-pocket = Pocket
+-brand-name-stripe = Stripe
+
+# The following are not terms because they are not used directly in messages,
+# but rather looked up in code and passed into the message as variables.
+product-name-apple-app-store = App Store
+product-name-google-play = Google Play
+
+## FxA product name to be used globally
+
+# "Accounts" can be localized and should be lowercase, "Firefox" must be treated as a brand.
```

Comment:
I don't think this comment is correct. "Firefox Accounts" was used to refer to the service, and was kept in English, unlike the reference to the actual account.

`main.ftl` only defines the term (uppercase), but it's never used.

`auth.ftl` has a couple of strings.

```
session-verify-send-push-title = Logging in to { -product-firefox-accounts }?
passwordChangeRequired-signoff-name = The { -product-firefox-accounts } team
```

The first should probably be changed to use the "Firefox account" term.

`settings.ftl` has the most strings (with the term still uppercase), mostly in the telemetry data section

```
app-default-title = { -product-firefox-accounts }
app-page-title = { $title } | { -product-firefox-accounts }
dc-subheader = Help improve { -product-firefox-accounts }
dc-subheader-content = Allow { -product-firefox-accounts } to send technical and interaction data to { -brand-mozilla }.
dc-opt-out-success = Opt out successful. { -product-firefox-accounts } won’t send technical or interaction data to { -brand-mozilla }.
dc-opt-in-success = Thanks! Sharing this data helps us improve { -product-firefox-accounts }.
```

Reply:
This comment was from auth.ftl:

```
# "Accounts" can be localized and should be lowercase, "Firefox" must be treated as a brand.
-product-firefox-accounts = Firefox accounts
```

auth.ftl also has 'Firefox account' marked as a brand, which might not be correct as it requires translation. Perhaps 'Firefox Accounts' should be marked as a term, but not 'Firefox account'?

Perhaps something like:
```
-product-firefox-accounts = Firefox Accounts
product-firefox-account = { -brand-name-firefox } account
```

Reply:
We have a separate ticket filed for this, [FXA-4087](https://mozilla-hub.atlassian.net/browse/FXA-4087).

Reply:
"Firefox account", as in one account, should definitely stay a term. Locales can define private variants with a term, and they're using it a lot for this specific case.